.travis.yml

```diff
@@ -5,6 +5,8 @@ matrix:
         - php: 5.3.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
         - php: 5.3
+        - php: 5.3
```

Comment:
They may be different. But you don't test all.
You test all PHP version with deps installed with ~some version, and then the lowest supported PHP version with lowest version of deps. Dont care about the rest.

If you would, why not test every single dep in every single version on every PHP engine version? It would create thousands of jobs.